### PR TITLE
Add Servant and the Cryptography Group to the Affiliates page

### PR DESCRIPTION
**Pull request by Kleidukos**
This PR adds Servant & the Cryptography Group to the list of affiliates.

For reference:

* https://github.com/haskell-cryptography/governance/issues/3

* https://github.com/haskell-servant/haskell-servant.github.io/issues/65

Once the core members of these groups have expressed their votes, this PR will be marked as ready.

### CODE REVIEW

**Comment by tomjaguarpaw** [comment]
It could be helpful from the point of view of auditability if this PR came with links to decisions or statements made by the maintainers of those projects (perhaps in the project READMEs) about their affiliation.

**Comment by Kleidukos** [comment]
@tomjaguarpaw Absolutely, I'll link to both then the approval can proceed.

**Comment by Kleidukos** [comment]
@tomjaguarpaw @myShoggoth The PR is ready, both groups have expressed their public approval.

**Comment by tomjaguarpaw** [comment]
That's great to hear! I think it would still be useful to leave links to the public approval here, so save people the time from searching for them.

Nevertheless, I don't think that absence should block merging the PR.

**Comment by Kleidukos** [comment]
@tomjaguarpaw Based on the other documents of affiliation in this repo I don't really see where we could put the public votes' links. But maybe you have something in mind?

**Comment by tomjaguarpaw** [comment]
Oh, I just meant leave them in this PR discussion thread.

**Comment by tomjaguarpaw** [comment]
I mean, I for one would like to see them, but I don't know where to look!

**Comment by tomjaguarpaw** [comment]
Oh I beg your pardon: I didn't see you had edited your original message to include those links.

**Review by tomjaguarpaw — APPROVED** [review]

**Comment by Kleidukos** [comment]
@tomjaguarpaw it's the "deploy" part of the CI that seems to return an exit code 1, do you think it's safe to merge?

**Comment by tomjaguarpaw** [comment]
I don't see anything that actually went wrong with Deploy, even though it finished with exit code 1. So I suspect it is safe to merge, but on the other hand I don't know how to fix it if something goes wrong, so perhaps best to wait until we find someone who does know.

**Comment by myShoggoth** [comment]
The issue is that the CI stuff doesn't work right if you're not merging into Hakyll.  !170 attempts to address this, but I haven't been able to verify that it works, and it changes a few core assumptions that need to be verified.

We fundamentally want CI to check the code vs. CI to deploy to be separate things, and only have the latter run on PRs to the Hakyll branch.

**Pull request merged**
